### PR TITLE
DOC: CLI and sig:// usage

### PR DESCRIPTION
**Pull request by klauer**
## Description
* Some documentation additions
* I didn't see these touched on elsewhere, but if I missed them, then please disregard

### CODE REVIEW

File: docs/source/basic_usage.rst [hunk 1]
```diff
@@ -64,8 +64,19 @@ Ophyd objects so that we can load them in a variety of contexts. If you do not
...
 
-Here is a quick example if you wanted to get a feel for what ``typhos`` looks
-like with `happi``:
+From the command-line, using typhos and happi together is easy.  For example,
+to load an auto-generated typhos screen for your device named ``"my_device"``
+would only require the following:
+
+.. code:: bash
+
+   $ typhos my_device
```
Comment (tangkong) [inline docs/source/basic_usage.rst:73]: I wonder how many people have typed `typhos load my_device` like me, after all the `happi load my_device`

**Review by tangkong — APPROVED** [review]
LGTM! Hooray for documentation.

**Review by ZLLentz — APPROVED** [review]
LGTM

**Comment by ZLLentz** [comment]
tests failing in specifically python 3.8 which is confusing, can't possibly be related to this PR

**Comment by klauer** [comment]
Tests failing due to pcdsdevices usage of Python 3.9+ type hints.  Fortunately, it's only a dev dependency (I think?)

**Comment by ZLLentz** [comment]
So, Python 3.7 is totally happy with Python 3.9+ type hints?

**Comment by klauer** [comment]
Definitely not. Several things at play here:
1. https://github.com/conda-forge/pcdsdevices-feedstock/issues/37
2. This `importorskip` allows 3.7 to skip:
https://github.com/pcdshub/typhos/blob/2c75e80a590f710c8dac7ae12e55bee07900cc09/typhos/tests/variety_ioc.py#L10
3. But 3.8 gets past the initial import and fails on the subsequent one

**Pull request merged**